client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1448,6 +1448,26 @@ Undocumented.prototype.usersEmailNew = function( query, fn ) {
 	return this.wpcom.req.post( args, fn );
 };
 
+/**
+ * Sign up for a new user account and login immediately
+ * Onboard a new user
+ *
+ * @param {object} query - an object with the following values: email
+ * @param {Function} fn - Function to invoke when request is complete
+ */
+Undocumented.prototype.usersEmailOnboard = function( query, fn ) {
+	debug( '/users/email/onboard' );
+
+	// This API call is restricted to these OAuth keys
+	restrictByOauthKeys( query );
+
+	const args = {
+		path: '/users/email/onboard',
```

Comment:
How does the passwordless onboarding work? What will be my username and password?

Reply:
Your username is auto-generated. 

Did you try the flow? You will have received an email with your username after the **Howdy.**

As for the password, Calypso will force you to use Magic Link login until you set a password yourself. There's an action item in the checklist.

One further task might be is to modify the email body to inform the user that they can choose a better username in settings.

Reply:
> Did you try the flow?

No 🙂 I just looked at the code. Thanks for explaining! I admit that I've never tried magic login either.